test/index-test.js

```diff
@@ -37,21 +55,19 @@ describe('Server', () => {
         expect(nonDefaultConfig).eql(expectedConfig);
     });
 
-    it(`Sends response from endpoint '/'`, async () => {
-        const expectedStatus = 200;
-
-        const response = await got(rootPath);
-
-        expect(response.statusCode).equal(expectedStatus);
-    });
+    it(`Listens on host and port provided in App.listen()`, async () => {
+        process.env.PORT = 'nondefault';
+        process.env.HOST = 'nondefault';
+        const constructorPort = '8000';
+        const constructorHost = '127.0.0.1';
 
-    it('Sends the correct html', async () => {
-        const expectedBody = 'Hello Node.js';
-        const bodyRegex = /<body.*?>([\s\S]*)<\/body>/;
+        await server.close();
```

Comment:
Let's an empty line after this line and replace the `newline-after-var` ESlint rule with the [padding-line-between-statements](https://eslint.org/docs/rules/padding-line-between-statements) rule. I guess we can try to use `*` for both `prev` and `next` rule configuration options.